### PR TITLE
Change Cron Binding from `input/output` to `input` binding

### PR DESCRIPTION
**Pull request by sarsharma**
# Description

Change cron binding to input only binding to certify it as a stable component

Docs PR : https://github.com/dapr/docs/pull/2971

## Issue reference
https://github.com/dapr/components-contrib/issues/2308

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -69,6 +69,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 )
 
+require github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.5.1 // indirect
```
Comment (ItalyPaleAle) [inline go.mod:72]: Why did this end up in its own block?

Comment (sarsharma) [reply go.mod]: Not sure, running `make modtidy-all` added it in this way. I have placed it in the correct block now.

**Review by mukundansundar — DISMISSED** [review]

**Comment by mukundansundar** [comment]
@sarsharma can you fix the linter errors please?

**Review by ItalyPaleAle — DISMISSED** [review]

**Review by ItalyPaleAle — APPROVED** [review]

**Review by artursouza — DISMISSED** [review]

**Comment by mukundansundar** [comment]
> lgtm.
> @dapr/maintainers-dapr note this is a breaking change for an alpha component. We are removing output binding completely and changing cron to only input binding.

relavant discussion https://github.com/dapr/components-contrib/pull/2191#discussion_r1016973242

**Review by ItalyPaleAle — APPROVED** [review]

**Pull request merged**